docs/tutorials/earthquake_detection.ipynb

```diff
@@ -0,0 +1,3452 @@
+    "with h5py.File('data/embeddings.h5', 'w') as f:\n",
+    "    # We iterate over the train_dataloader\n",
+    "    for i, batch in tqdm(enumerate(datamodule.train_dataloader()), desc='Train'):\n",
+    "        for j, (emb, lab) in enumerate(zip(*infer(batch))):\n",
+    "            f.create_dataset(f'train/{i}_{j}', data=emb, compression='gzip')\n",
+    "            f[f'train/{i}_{j}'].attrs['label'] = lab\n",
+    "    # We iterate over the test_dataloader\n",
+    "    for i, batch in tqdm(enumerate(datamodule.test_dataloader()), desc='Test'):\n",
+    "        for j, (emb, lab) in enumerate(zip(*infer(batch))):\n",
+    "            f.create_dataset(f'test/{i}_{j}', data=emb, compression='gzip')\n",
+    "            f[f'test/{i}_{j}'].attrs['label'] = lab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can load the embeddings from the file and fit a classical model (i.e., Random Forest) using the embeddings as features and the labels as targets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "executionInfo": {
+     "elapsed": 7180,
+     "status": "ok",
+     "timestamp": 1742053727478,
+     "user": {
+      "displayName": "Daniele Rege Cambrin (Darth Reca)",
+      "userId": "08456786386580134960"
+     },
+     "user_tz": -60
+    },
+    "id": "9q8OqFoHt3-A",
+    "outputId": "3fdba6f0-3d6a-49e5-8d07-46aacbaf04e3"
+   },
+   "outputs": [],
+   "source": [
+    "embeddings = defaultdict(list)\n",
+    "labels = defaultdict(list)\n",
+    "with h5py.File('data/embeddings.h5', 'r') as f:\n",
+    "    for split in ['train', 'test']:\n",
+    "        for key in f[split]:\n",
+    "            embeddings[split].append(f[split][key][...])\n",
+    "            labels[split].append(f[split][key].attrs['label'])\n",
+    "embeddings = {k: np.stack(v) for k, v in embeddings.items()}\n",
+    "labels = {k: np.array(v) for k, v in labels.items()}\n",
+    "# Train a RandomForest classifier\n",
+    "clf = RandomForestClassifier(n_estimators=100, n_jobs=-1)\n",
+    "clf.fit(embeddings['train'], labels['train'])\n",
+    "# Evaluate the classifier on the test set\n",
+    "preds = clf.predict(embeddings['test'])\n",
+    "print(classification_report(labels['test'], preds))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training a deep model from scratch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hQITTSEEt3-A"
+   },
+   "source": [
+    "The second approach requires training a deep neural network from scratch. To this end, we can use the TorchGeo's ```ClassificationTask``` and Lightning's ```Trainer``` to simplify the training.\n",
+    "\n",
+    "Remember to set the ```in_channels``` parameter to 4 since we are concatenating two (pre and post-event) two-channels images."
```

Comment:
```suggestion
    "Remember to set the ```in_channels``` parameter to 4 since we are concatenating two (pre and post-event) two-channel images."
```